src/Orleans/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "dependencies": {
-    "Newtonsoft.Json": "7.0.1",
+    "Newtonsoft.Json": "9.0.1",
```

Comment:
what prompted this change? If this is needed, let's do it in a separate PR, as there are a lot of side effects and stuff that needs to be updated for this (ie: in the nugets, VSO, and also samples in the future, but we can worry about the latter later)